hazelcast/src/test/java/com/hazelcast/cache/eviction/CacheExpirationTest.java

```diff
@@ -77,10 +85,10 @@
     @Parameterized.Parameter(0)
     public boolean useSyncBackups;
 
-    private final Duration FIVE_SECONDS = new Duration(TimeUnit.SECONDS, 5);
+    private final Duration THREE_SECONDS = new Duration(TimeUnit.SECONDS, 3);
```

Comment:
Is this going to make the tests more timing sensitive (flaky)?

Reply:
lets see.